gemrb/core/System/Logger.h

```diff
@@ -27,11 +27,28 @@
 
 #include "exports.h"
 
-#include "System/Logging.h"
+#include <atomic>
+#include <condition_variable>
+#include <deque>
+#include <memory>
+#include <mutex>
+#include <string>
+#include <thread>
 
 namespace GemRB {
 
-enum log_color {
+// !!! Keep this synchronized with GUIDefines !!!
+enum log_level : int {
+	INTERNAL = -1, // special value that can only be used by the logger itself. these messages cannot be supressed
+	FATAL = 0,
+	ERROR = 1,
```

Comment:
This seems to be conflicting with some `ERROR` enum value in `windows.h`. `enum class` to the rescue?

Reply:
if `enum class` does the job, sure. Else there is always scoping it. However, `GemRB::log_level::ERROR` is already distinct from `::ERROR`. What exactly is the error? A symbol ambiguity error?

Reply:
```
D:/Sources/gemrb/gemrb/core/System/Logger.h:44:2: error: expected identifier before numeric constant
   44 |  ERROR = 1,
      |  ^~~~~
```
First thought was these C enum tokens overlap. But actually it looks like it's a macro from inside of the Windows headers that can be suppressed by `#define NOGDI` right before loading `windows.h`. I tested this and it works.

Reply:
Wait before applying this, there something else down the line in `Win32Console.cpp` and I'm not sure yet wheter this related or independent from this workaround. Will check later.

Reply:
If that's problematic, an undef should work.